drivers/gles3/storage/material_storage.cpp

```diff
@@ -2979,9 +2979,11 @@ void SceneShaderData::set_code(const String &p_code) {
 		WARN_PRINT_ONCE_ED("Transmittance is only available when using the Forward+ rendering backend.");
 	}
 
+	/* TODO I think this warning is a left over, need to discuss with Clay and if correct, remove!
```

Comment:
@clayjohn is this warning a leftover? as far as I can tell depth texture is working. Or does it only work in certain conditions?

Reply:
Yes, please remove

Reply:
Done